src/libraries/Microsoft.PowerFx.Core/Binding/INameResolver.cs

```diff
@@ -82,6 +87,9 @@ internal interface INameResolver
         bool LookupGlobalEntity(DName name, out NameLookupInfo lookupInfo);
 
         bool TryLookupEnum(DName name, out NameLookupInfo lookupInfo);
+
+        // Look up a type by name.
+        bool LookupType(DName name, out FormulaType fType);
```

Comment:
new interface method means breaking internals. 
Will need PA update to absorb

Reply:
yes, I have a PA side PR ready